packages/react-sdk/docusaurus/docs/React/02-guides/03-call-and-participant-state.mdx

```diff
@@ -3,89 +3,193 @@ title: Call & Participant State
+Then you'll be able to use the [client state hooks](../../call-engine/hooks-and-contexts/#client-state) to easily access the client state anywhere in your application. These hooks are updated when a relevant client state change occurs.
 
 Let's see an example:
 
-If you want to watch for incoming calls to display a notification you can use the `useCalls` hook.
+If you want to observe the connected user you can use the [`useConnectedUser`](../../call-engine/hooks-and-contexts/#useconnecteduser) hook.
 
-TODO give example for `useCalls` once the refactor is done
+```tsx
+import {
+  useCreateStreamVideoClient,
+  StreamVideo,
+  useConnectedUser,
+} from '@stream-io/video-react-sdk';
+
+export default function App() {
+  const client = useCreateStreamVideoClient({
+    apiKey: '<Your API key>',
+    tokenOrProvider: '<token provider>',
+    user: {
+      id: '<user id>',
+      name: '<name>',
+    },
+  });
+  return (
+    <StreamVideo client={client}>
+      <Header></Header>
+    </StreamVideo>
+  );
+}
+
+const Header = () => {
+  const user = useConnectedUser();
+  return <div>{user ? `Logged in: ${user.name}` : 'Logged out'}</div>;
+};
+```
 
 This approach makes it possible to access the client state and be notified about changes anywhere in your application without having to manually subscribe to WebSocket events.
 
 The full list of available client state hooks are available in the [Hooks and Contexts page](../../call-engine/hooks-and-contexts#client-state).
 
 ## Observe call state
 
-To observe call state you need to use the [`StreamCallProvider`](../../call-engine/hooks-and-contexts#streamcallprovider):
+To observe call state you can use [`StreamMeeting` component](TODO link):
 
 ```tsx
-const client = useCreateStreamVideoClient({
-    <your API key>,
-    <user token>,
-    <user>,
-});
-
-const activeCall = // this should change with the refactor
-
-<StreamVideo client="{client}">
-  <StreamCallProvider call="{activeCall}">
-    <!-- Video UI -->
-  </StreamCallProvider>
-</StreamVideo>
+import {
+  useCreateStreamVideoClient,
+  StreamVideo,
+  useConnectedUser,
+} from '@stream-io/video-react-sdk';
+
+export default function App() {
+  const client = useCreateStreamVideoClient({
+    apiKey: '<Your API key>',
+    tokenOrProvider: '<token provider>',
+    user: {
+      id: '<user id>',
+      name: '<name>',
+    },
+  });
+
+  return (
+    <StreamVideo client={client}>
+      <Header></Header>
+      <StreamMeeting
+        callType="default"
+        callId="test-call"
+        autoJoin={true}
+        data={{ create: true }}
+      ></StreamMeeting>
+    </StreamVideo>
+  );
+}
+
+const Header = () => {
+  const user = useConnectedUser();
+  return <div>{user ? `Logged in: ${user.name}` : 'Logged out'}</div>;
+};
```

Comment:
```suggestion
  return (
    <StreamVideo client={client}>
      <StreamMeeting
        callType="default"
        callId="test-call"
        autoJoin={true}
        data={{ create: true }}
      >
        <Header />
      </StreamMeeting>
    </StreamVideo>
  );
}

const Header = () => {
  const user = useConnectedUser();
  const recording = useIsCallRecordingInProgress()
  return (
    <div>
      { recording && <RecordingBadge />}
      {user ? `Logged in: ${user.name}` : 'Logged out'}
    </div>
  );
};
```

Reply:
The next step will introduce the `useCall` and `useCallingState` hooks, I can also include the `useIsCallRecordingInProgress` hook if you want to, but I'd rather add it to the `Call` component